src/test/java/bio/terra/service/snapshotbuilder/query/QueryTest.java

```diff
@@ -25,4 +28,89 @@ public static Query createQuery() {
+            List.of(
+                new FieldVariable(
+                    new FieldPointer(tablePointer, "person_id", "COUNT"),
+                    tableVariable,
+                    null,
+                    true)),
+            List.of(tableVariable, conditionOccurrenceVariable, conditionAncestorVariable),
+            new BooleanAndOrFilterVariable(
+                BooleanAndOrFilterVariable.LogicalOperator.AND,
+                List.of(
+                    new BooleanAndOrFilterVariable(
+                        BooleanAndOrFilterVariable.LogicalOperator.OR,
+                        List.of(
+                            new BinaryFilterVariable(
+                                new FieldVariable(
+                                    new FieldPointer(
+                                        conditionOccurrencePointer, "condition_concept_id"),
+                                    conditionOccurrenceVariable),
+                                BinaryFilterVariable.BinaryOperator.EQUALS,
+                                new Literal(316139)),
+                            new BinaryFilterVariable(
+                                new FieldVariable(
+                                    new FieldPointer(
+                                        conditionAncestorPointer, "ancestor_concept_id"),
+                                    conditionAncestorVariable),
+                                BinaryFilterVariable.BinaryOperator.EQUALS,
+                                new Literal(316139)),
+                            new BinaryFilterVariable(
+                                new FieldVariable(
+                                    new FieldPointer(
+                                        conditionOccurrencePointer, "condition_concept_id"),
+                                    conditionOccurrenceVariable),
+                                BinaryFilterVariable.BinaryOperator.EQUALS,
+                                new Literal(4311280)),
+                            new BinaryFilterVariable(
+                                new FieldVariable(
+                                    new FieldPointer(
+                                        conditionAncestorPointer, "ancestor_concept_id"),
+                                    conditionAncestorVariable),
+                                BinaryFilterVariable.BinaryOperator.EQUALS,
+                                new Literal(4311280)))),
+                    new BinaryFilterVariable(
+                        new FieldVariable(
+                            new FieldPointer(tablePointer, "year_of_birth"), tableVariable),
+                        BinaryFilterVariable.BinaryOperator.LESS_THAN,
+                        new Literal(1983)))));
+    String querySQL = query.renderSQL(null);
+    assertThat(querySQL, containsString("SELECT COUNT(DISTINCT p.person_id) FROM person AS p"));
+    assertThat(
+        querySQL, containsString("JOIN condition_occurrence AS c ON c.person_id = p.person_id"));
```

Comment:
If you want to assert many things about one value, you can do it using `allOf()`.
```suggestion
    assertThat(querySQL, allOf(containsString("SELECT COUNT(DISTINCT p.person_id) FROM person AS p"), 
    containsString("JOIN condition_occurrence AS c ON c.person_id = p.person_id"));
```